bin/cronjob

```diff
@@ -11,14 +11,14 @@
 #   or returned output.
 #
 # ERROR (not currently differentiated): A non-0 exit code could be elevated to mean error and
-#   result in a red HipChat message and/or channel notification.
+#   result in a red Slack message and/or channel notification.
 #
 # CRONJOBs are defined the cdo-apps::crontab Chef recipe: `chef/cookbooks/cdo-apps/recipes/crontab.rb`.
 #
 
 require_relative '../deployment.rb'
 require 'mail'
-require 'cdo/hip_chat'
+require 'cdo/chat_client'
```

Comment:
FWIW we removed cronjob hipchat logging in favor of honeybadger in https://github.com/code-dot-org/code-dot-org/pull/4426. It should be safe to remove this require, and verify `cronjob` isn't broken since is small and doesn't have many branches.